admin/src/js/controllers/upgrade.js

```diff
@@ -223,9 +223,9 @@ angular.module('controllers').controller('UpgradeCtrl',
       return $http
         .post(url, { build })
         .catch(err => {
-          // todo which status do we get with nginx???
           // exclude "50x" like statuses that come from nginx
-          if (err && (!err.status || err.status === 503 || err.status === -1) && action === 'complete') {
+          const requestFailedStatuses = [1, 502, 503];
```

Comment:
Did you mean to change `-1` to `1` here?  (I am not really sure what the case might be where the status would be either value, but seemed like the change might not have been intentional....)

Reply:
oh! good spotting. It's supposed to be -1!